packages/webapp/src/hooks/query/authentication.tsx

```diff
@@ -37,6 +38,64 @@ export const useAuthLogin = (props) => {
   });
 };
 
+
+/**
+ * Authentication OIDC authorize
+ */
+
+export const useAuthOidcAuthorize = (props) => {
+  const apiRequest = useApiRequest();
+
+  return useMutation((values) => apiRequest.post('oidc/authorize', values), {
+    select: (res) => res.data,
+    onSuccess: (data) => {
+      window.location.href = data.data.authorization_url;
+    },
+    ...props,
+  });
+};
+
+/**
+ * Authentication OIDC login
+ */
+
+export const useAuthOidcLogin = (props) => {
+  const apiRequest = useApiRequest();
+
+  return useMutation((values) => apiRequest.post('oidc/login', values), {
+    select: (res) => res.data,
+    onSuccess: (data) => {
+      // Set authentication cookies.
+      setAuthLoginCookies(data.data);
+
+      window.location.href = '/';
```

Comment:
@abouolia Previously while logging in, the login page was reloaded after the successful login. Here in oidc login we are supposed to get code as query param in login url so we are redirecting it to '/' which is a bit static for now as reloading the route will make api call for authorization callback. Do we have any standard path that needs to be added here after login or what will be the best use case here?